include/gz/common/Console.hh

```diff
@@ -27,24 +27,30 @@
 #include <gz/common/Export.hh>
 #include <gz/utils/SuppressWarning.hh>
 
-namespace ignition
+namespace gz
 {
   namespace common
   {
+    // TODO(CH3): Remove all the ign macros on tock
```

Comment:
I like to leave a comment that has the word "deprecated" on it so it's easier to find when performing the tock. Otherwise this may get lost in the pool of TODOs

Reply:
Done! https://github.com/gazebosim/gz-common/pull/356/commits/dcc40dce6630288ba9ed67b08a0bd08499be3404 

I also did so in `gz-sim`